eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -127,6 +128,8 @@ trait Eclair {
 
   def onChainBalance()(implicit timeout: Timeout): Future[OnChainBalance]
 
+  def listTransactions(count: Int, skip: Int)(implicit timeout: Timeout): Future[Iterable[WalletTransaction]]
```

Comment:
For consistency with `onChainBalance`:

```suggestion
  def listOnChainTransactions(count: Int, skip: Int)(implicit timeout: Timeout): Future[Iterable[WalletTransaction]]
```

or just `onChainTransactions`

Reply:
Done in https://github.com/ACINQ/eclair/pull/1461/commits/80b23340a99b886b6df1df79551b4c2b33ab093a